src/commands/build.js

```diff
@@ -2,12 +2,13 @@ import fs from "fs-extra";
 import path from "path";
 import webpack from "webpack";
 import getWebpackConfig from "../config/getWebpackClientConfig";
+import generateStaticHTMLFile from "../lib/server/generateStaticHTMLFile";
```

Comment:
You need to run it with `NODE_ENV=production` which will make it so that hot module replacement plugins are not applied